eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/immutable/ImmutableArrayBag.java

```diff
@@ -246,6 +246,25 @@ public T getLast()
     }
 
     @Override
+    public T getOnly()
+    {
+        Iterator<T> iterator = this.iterator();
+
```

Comment:
The reason why I override getOnly() at `ImmutableArrayBag`, I thought `size()` is O(n).

```java
    @Override
    public int size()
    {
        int sum = 0;
        for (int value : this.counts)
        {
            sum += value;
        }
        return sum;
    }
```

So I thought this class should not use `size()` method. Like `AbstractLazyIterable`.